htmresearch/frameworks/nlp/htm_runner.py

```diff
@@ -297,6 +299,15 @@ def _selectWinners(self, predictions):
       for p in predictions:
         counter.update(p)
       return zip(*counter.most_common(self.numClasses))[0]
+    elif self.votingmethod == "tree":
+      import pdb; pdb.set_trace()
```

Comment:
Sorry, looks like this and some other changes in the runners got looped into this PR by mistake.